docs/_components/modal.md

```diff
@@ -9,34 +9,36 @@ Modals are a blocking element, they should be used sparingly and only when we ab
 Example:
 
 ```html
-<div class="modal">
+<div class="modal" id="myModal" role="dialog" aria-labelledby="dialog-title" aria-describedby="dialog-description">
     <div class="modal__dialog">
         <div class="modal__content">
             <div class="modal__header">
-                <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
-                <h4 class="modal__title">A simple example</h4>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close dialog">×</button>
+                <h4 class="modal__title" id="dialog-title">A simple example</h4>
             </div>
             <div class="modal__body">
+                <p id="dialog-description" class="sr-only">SR description goes here</p>
                 <p>The modal body might have instructions, a form, or other stuff.</p>
             </div>
             <div class="modal__footer">
-                <button type="button" class="btn" data-dismiss="modal">Cancel</button>
                 <button type="button" class="btn btn--primary">Save Changes</button>
+                <button type="button" class="btn" data-dismiss="modal">Cancel</button>
             </div>
         </div><!-- /.modal__content -->
     </div><!-- /.modal__dialog -->
 </div><!-- /.modal -->
 ```
 
 <div class="pulsar-example">
-    <div class="modal modal__example show">
+    <div class="modal modal__example show" id="myModal2" role="dialog" aria-labelledby="dialog2-title" aria-describedby="dialog2-description">
         <div class="modal__dialog">
             <div class="modal__content">
                 <div class="modal__header">
-                    <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
-                    <h4 class="modal__title">A simple example</h4>
+                    <button type="button" class="close" data-dismiss="modal" aria-label="Close dialog">×</button>
+                    <h4 class="modal__title" id="dialog2-title">A simple example</h4>
                 </div>
                 <div class="modal__body">
+                    <p id="dialog2-description" class="sr-only">SR description goes here</p>
```

Comment:
same as above, no `sr-class`.

Reply:
The use of `dialog` in the id attributes is confusing, I'd still recommend we change

`dialog2-title` to `myModal2-title`
`dialog2-description` to `myModal2-description`

It feels more obvious for developers to update and maintain based on the parent modal id.